examples/pennylane_run_isingmodel_PyTorch.py

```diff
@@ -0,0 +1,151 @@
+r"""
+.. _isingmodel_PyTorch:
+
+Optimization of 3-qubit Ising model in PyTorch
+===============================================
+
+The Ising model is an example of a system whose optimization landscape is non-convex. Hence, using gradient descent may not be the best strategy as the optimization can get
```

Comment:
Do you want to add a link to the wikipedia page on Ising models here?

Reply:
ah yes! good idea!